packages/cardpay-reward-api/cardpay_reward_api/models.py

```diff
@@ -28,6 +28,6 @@ class Root(Base):
     id = Column(Integer, primary_key=True, index=True)
     rewardProgramId = Column(String)
     rootHash = Column(String, unique=True)
-    paymentCycle = Column(Integer)
+    paymentCycle = Column(Integer, unique=True)
     blockNumber = Column(Integer)
-    timestamp = Column(Time)
+    timestamp = Column(DateTime)
```

Comment:
Some mistakes here in the model. Will need to clear out dbs during deploy

Reply:
Worth just moving to sqlite?

Reply:
I dont see why not. I looked a bit. it shud be faster and ecs filesystem shud support. 

But, there is some procedures that I need to figure out (like how to tunnel). did u do work on that previously?